sklearn/utils/__init__.py

```diff
@@ -599,3 +599,39 @@ def is_scalar_nan(x):
     # Redondant np.floating is needed because numbers can't match np.float32
     # in python 2.
     return bool(isinstance(x, (numbers.Real, np.floating)) and np.isnan(x))
+
+
+def get_feature_importances(estimator, norm_order=1):
```

Comment:
we still might want to name this `_get_feature_importances`, otherwise we are promising API stability to users.

Reply:
Sure, will change it back to `_get_feature_importances`. Didn't think of that. 
Also, should I write more tests to pass codecov? (Currently it's 84.21%)